proposals/0020-hlsl-202x-202y.md

```diff
@@ -68,8 +75,60 @@ have workarounds and may pose a barrier to adoption.
 
 ### HLSL 202y
 
-HLSL 202y shall contain features that expand HLSL. This is all other outstanding
-feature proposals.
+HLSL 202y is an opportunity to address some frequently requested features that
+we can deliver more quickly in Clang than in DXC. Some of the features in HLSL
+202y require significant changes to core language behaviors and will likely
+cause source compatibility issues.
+
+Two big areas of investment in HLSL 202y are overload resolution and object
+initialization. While the two areas are intertwined, we will discuss them mostly
+separately.
+
+#### Overload Resolution
+
+In Clang we've adopted overload resolution behavior more similar to C++. This
+causes some source incompatibilities as cited in the [clang
+docs](https://clang.llvm.org/docs/HLSL/ExpectedDifferences.html). Taking these
+behavior changes into HLSL makes HLSL's overload best-match algorithm more
+familiar and will produce ambiguous resolution errors instead of potentially
+choosing the wrong overload when ambiguity exists in the source.

```

Comment:
> choosing the wrong overload when 

I think "wrong" is kind-of subjective.  Maybe "surprising" or "unexpected" or something about inconsistent choices?